test/6_branch.js

```diff
@@ -607,11 +607,20 @@ describe('Branch', function() {
 			requests[0].callback(null, expectedResponse);
 			assert.deepEqual(requests[0].obj, expectedRequest, 'All params sent');
 		});
+
+		it('should update identity and identity_id in local storage', function(done) {
+			var branch = initBranch(true);
+			var assert = testUtils.plan(2, done);
+			branch.setIdentity('12345678', function(err, data) {
+				var localData = safejson.parse(localStorage.getItem('branch_session_first'));
+				assert.strictEqual(localData['identity'], '12345678');
```

Comment:
@aaaronlopez This ensures that we pick up the right identity & identity_id at initialization.

Reply:
👍